components/insight/SRC/org/openmicroscopy/shoola/util/roi/io/InputServerStrategy.java

```diff
@@ -526,6 +528,17 @@ private void addShapeSettings(ROIFigure figure, ShapeSettingsData data)
 		FONT_ITALIC.set(figure, data.isFontItalic());
 		FONT_BOLD.set(figure, data.isFontBold());
 		TEXT_COLOR.set(figure, data.getStroke());
+		
+        if (CommonsLangUtils.isNotBlank(data.getMarkerStart())) {
+            if (data.getMarkerStart().equalsIgnoreCase("Arrow"))
```

Comment:
Created an Enum for that instead, makes it easier to add more line decorations if needed.